Package.swift

```diff
@@ -24,6 +27,21 @@ let package = Package(
         .target(name: "Runner", dependencies: ["RunnerLib", "MarathonCore", "Logger"]),
         .testTarget(name: "DangerTests", dependencies: ["Danger"]),
         .testTarget(name: "RunnerLibTests", dependencies: ["RunnerLib"]),
-    ],
-    swiftLanguageVersions: [4]
+    ]
 )
+
+#if canImport(PackageConfig)
+    import PackageConfig
+
+    let config = PackageConfig([
+        "komondor": [
+            "pre-push": "swift test",
+            "pre-commit": [
```

Comment:
Would be cool to add here
`swift test --generate-linuxmain`

Reply:
Then we can remove the travis thing that checks if the checks are synced, or just put `swift test --generate-linuxmain` after the swiftlint call to avoid that it fails, and keep it there, because it could still fail if a person didn't call the `komondor init`